api/reference/reports.md

```diff
@@ -86,6 +88,47 @@ Example for running a report on the following:
 }
 ```
 
+### Scheduling Reports
+
+Scheduling a report is done via the **schedule** action on a report resource as follows
+
+``` data
+POST /api/reports/:id
+```
+
+``` data
+{
+  "action" : "schedule"
+  "resource": {
+    "send_email": true|false,
+    "start_date": DateTime,
+    "time_zone":  TZ,
+    "interval": {
+      "unit":  "minutely"|"hourly"|"daily"|"weekly"|"monthly"|"once",
+      "value": "once"
+    },
+    ...
+  }
+}
+```
+
+Example for scheduling a "one off" report:
+
+``` data
+{
+  "resource": {
+    "name":        "Chargeback Report Schedule - 2000/01/01",
+    "description": "Chargeback Report 2000 New Year Report",
+    "resource_type": "MiqReport",
+    "start_date": "2000-01-01T00:00:00Z",
+    "time_zone": "UTC",
+    "interval": { "unit": "once" },
+    "enabled": true
+  }
+}
+```
+
+
```

Comment:
this new section should go below the "Running Reports" section, probably right before "Requesting Report Result Downloads" around line 363.   Also, maybe add the Request label and show the related Response. Thanks.